source/Detector/Amptek/AmptekSDD123Server.h

```diff
@@ -0,0 +1,133 @@
+#ifndef AMPTEKSDD123SERVER_H
+#define AMPTEKSDD123SERVER_H
+
+#include <QObject>
+#include <QTime>
+#include <QStringList>
+#include <QQueue>
+
+#include <QSharedMemory>
+#include <QTimer>
+
+#include "AmptekSDD123Packet.h"
+#include "AmptekSDD123ConfigurationMap.h"
+
+class QUdpSocket;
+
+class AmptekSDD123Server : public QObject
+{
+Q_OBJECT
+
+public:
+	AmptekSDD123Server(AmptekSDD123ConfigurationMap *configurationMap, const QString &serverName, QObject *parent = 0);
+
+	bool detectorUnavailable() const;
+
+	QList<double> droppedPacketStatistics() const;
+	QList<double> unrequestedPacketStatistics() const;
+
+	bool event(QEvent *e);
+
+public slots:
+	/// Returns id of newly created packet or -1 in case of failure
+	int forwardDatagram(const QByteArray &datagram);
+
+	void requestDataPacket(AmptekCommandManagerSGM::AmptekCommandDefSGM command, const QString &dataStringHex="", bool fakeSendRequest=false, bool sendSyncRequest=false, int overrideTimeout=-1);
+//	void requestStatus();
```

Comment:
These commented codes/functions will be deleted before merge.

I created a function ```requestDataPacket()``` to replace all those functions. I keep those replaced codes here for the convenience of reviewers to understand the functionality of ```requestDataPacket()```.